VVResonances/interactive/TailSmoothing3D.py

```diff
@@ -0,0 +1,173 @@
+import ROOT
```

Comment:
Is this file used anywhere? If not, I would remove it and keep it in your local code if you need it